torch/csrc/jit/register_distributed_ops.cpp

```diff
@@ -23,6 +28,29 @@ at::Tensor optional_to_tensor(c10::optional<at::Tensor> v) {
   return v.has_value() ? *v : at::Tensor();
 }
 
+c10::OperatorOptions aliasAnalysisFromSchema() {
+  c10::OperatorOptions result;
+  result.setAliasAnalysis(c10::AliasAnalysisKind::FROM_SCHEMA);
+  return result;
+}
+
+RegisterOperators reg_rpc_ops(
+    {Operator(
+         "aten::to_here(RRef(t) self) -> t",
+         [](Stack& stack) {
+           auto rref = pop(stack).toRRef();
+           IValue res;
+           if (rref->isOwner()) {
+              res = c10::dynamic_intrusive_pointer_cast<dist_rpc::OwnerRRef>(rref)->getValue();
+           } else {
+              res = c10::dynamic_intrusive_pointer_cast<dist_rpc::UserRRef>(rref)->toHere();
```

Comment:
now toHere() is rawIValue, need to call res = std::move(rawValues).front() after this line

Reply:
I changed the UserRRef C++ API to still return a single IValue instead of a std::vector<IValue>. IValue should itself handle the list abstraction.